src/components/video-controller.js

```diff
@@ -379,36 +517,31 @@ export class VideoController {
                     return;
                 }
             } else if (!adBreak.started) {
-                this.startAd(adBreak);
+                // We will get ad events when the ad is encountered
```

Comment:
how about we add "...ad events _from Google IMA stream event_..."?

Reply:
ok, will fix